digitalfabrik/entitlementcard
# 1692: Add this card has been revoked error message

### Short description
Add a separate error message when the user tries to activate a revoked card in the app

### Proposed changes
- add additional activation state = revoked & corresponding error message
- I also renamed the QrParsingErrorDialog to ActivationErrorDialog and updated the title of the dialog to make it more generic, since it is not only used when scanning code, but also in the deep link activation

### Side effects
no

### Testing
1. create any card (for any project)
2. update revoked = true in the database
3. try to activate the revoked card via qr code scanning & via deep link activation
--> **"This card could not be activated as it has been revoked." error message should be displayed**

### Resolved issues
Fixes: #1692
